components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/java/org/wso2/carbon/identity/webhook/metadata/api/constant/ErrorMessage.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.webhook.metadata.api.constant;
+
+/**
+ * Error messages for webhook metadata.
+ */
+public enum ErrorMessage {
+
+    // Client errors
+    INVALID_REQUEST("CLIENT-00001", "Invalid request",
+            "The request is invalid."),
+    PROFILE_NOT_FOUND("CLIENT-00002", "Profile not found",
+            "The requested event profile could not be found."),
+    SCHEMA_NOT_FOUND("CLIENT-00003", "Schema not found",
+            "The requested schema could not be found."),
```

Comment:
Let's use one word, either profile or schema. I think we agreed to profile

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6782/commits/13dd6b0be593c982ff72e2b4116143ce3c5cf6e8

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6782/commits/642bd29a9d6a52e4e0b68a9eaf20cdc8cfcb4ae0